Makefile

```diff
@@ -24,9 +24,9 @@ IMAGE_PULL_POLICY ?= Always
 lint: lint-go-mod lint-modified-files
 
 lint-go-mod:
-	go mod verify
+	go mod tidy
 
-lint-modified-files:
+lint-modified-files: | lint-go-mod
```

Comment:
TIL about `|` in makefile targets.

Reply:
It's like that every day working with Mark. :)